client/my-sites/site-settings/reading-site-settings/YourHomepageDisplaysSetting.tsx

```diff
@@ -1,5 +1,3 @@
-import { useLocale, localizeUrl } from '@automattic/i18n-utils';
```

Comment:
We still need `localizeUrl` imported - as it is used on line `163`. Otherwise we are getting `Uncaught ReferenceError: localizeUrl is not defined` error. 🙂

Reply:
TIL that GH doesn't allow applying change suggestions on removed lines:
![Markup on 2023-02-10 at 16:48:00](https://user-images.githubusercontent.com/25105483/218134771-da830602-fd72-4e45-a87c-63c6a0ab8585.png)